packages/nvidia_nat_strands/src/nat/plugins/strands/strands_callback_handler.py

```
@@ -0,0 +1,489 @@
+                data=StreamEventData(input=str(tool_input), output=""),
+                metadata=TraceMetadata(
+                    tool_inputs=copy.deepcopy(tool_input),
+                    tool_info=copy.deepcopy(getattr(selected_tool, 'tool_spec', {})),
+                ),
+                usage_info=UsageInfo(token_usage=TokenUsageBaseModel()),
+            )
+
+            step_manager.push_intermediate_step(start_payload)
+
+            logger.debug("TOOL_START: %s (ID: %s)", tool_name, tool_use_id)
+        except Exception:  # noqa: BLE001
```

Ruff (0.14.2)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)